### ISSUE TITLE
Packages used in tutorial are missing

### ISSUE THREAD

**Issue by Screenium** [analysiscenter/radio#5]
Hello, 

I am not able to implement any of theese packages from the tutorial: 

from radio.dataset import FilesIndex, Pipeline
from utils import show_slices

I followed the installation instructions but theese seems just missing. Are those turorial outdated?

As well I would like to ask you if its possible to use LIDR-IDRI dataset instead of LUNA16 (considering different metadata files included marking the nodles as I was not able to look inside the FilesIndex package)

Thank you.

**Comment by jwitos** [comment]
Hi,

Please make sure you do have the correct package installed, for example running `pip install radio` gets you the wrong one, you can check it by calling `pip show radio`.

To get RadIO to load I had to run `pip install git+https://github.com/analysiscenter/radio.git`

Cheers

**Comment by Screenium** [comment]
Hi,

I figured that the moment I used pip for the first time. I then removed the random radio library and get the right one from git repository. `Pip show radio` gives me the right result. But I am still missing the mentioned parts of the package.

Thank you for answer anyway.

**Comment by Screenium** [comment]
UPDATE: 
Even though I got installed full packages with the dataset I downloaded the stand alone dataset library and imported the packages from there. So far so good. 

Utils still missing. 

New problem: 'lunaset.train >> crop_pipeline).run()' just freezes my PC for eternity. I have made new subset of just 10 .mhd images but my PC is still crashing at that point. Restarts even (after long time of being freezed). Running on win10 with i7 processor. Any one experiencing the same problem? 

Thanks
